src/types/index.ts

```
@@ -0,0 +1,52 @@
+// イベント型
+export interface AppEvents {
+  "open-dice-modal": void;
+  "reset-state-notification": void;
+  "navigate-to-simu": void;
+  "navigate-to-tcg": void;
+  "update-main-dice-from-child": number;
+}
+
+export interface DiceModalEvents {
+  close: void;
+  "roll-complete": number[];
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)